tf_encrypted/__init__.py

```diff
@@ -6,8 +7,7 @@
 from .config import Config, LocalConfig, RemoteConfig, get_config
 from .player import player
 from .protocol import Pond, get_protocol
-from .session import (Session, setTFEDebugFlag, setMonitorStatsFlag,
-                      setTFETraceFlag, setLogDirectory)
+from .session import Session, set_tfe_debug_flag, set_tfe_events_flag, set_tfe_trace_flag, set_log_directory
```

Comment:
this line seems too long

Reply:
TensorFlow explicitly allows longer import statements because of the depth of their package tree.  You're right though, this seems unnecessary in this case.